File: Modul 4/soal 4/PRAK304-2210817310014-Fatih Alif Ghifari.py
i=0
while True:
    i=i+1
    nilai=int(input())
    if(nilai>=1 and nilai<=9):
        print("Satuan\n")
    elif(nilai==0):
        print("Nol\n")
    elif(nilai>=99):
        print("Anda Menginput Melebihi Limit Bilangan\n")
    elif(nilai>=20 and nilai<=99):
        print("Puluhan\n")
    else:
        print("Belasan")
    if(i>4):
        break

File: Modul4/soal4/PRAK304-2210817310014-FatihAlifGhifari.c
#include<stdio.h>

int main ()
{
    int in,i;
    while (i<=4) {
        printf ("-->");
        scanf ("%d", &in);
        if (in>0&&in<10){
            printf("Satuan\n");
        }
        else if (in>10&&in<20){
            printf("Belasan\n");
        }
        else if (in>=10&&in<100){
            printf("Puluhan\n");
        }
        else if (in==0){
            printf("Nol\n");
        }
        else if (in>99){
            printf("Anda Menginput Melebihi Limit Bilangan\n");
        }
        i++;
    }
    return 0;
}
